src/modules/LanItems.rb

```diff
@@ -2636,6 +2653,13 @@ def enslaved?(ifcfg_name)
 
   private
 
+    # Return the current name of the {LanItem} given
+    #
+    # @param item_id [Integer] a key for {#Items}
+    def current_name_for?(item_id)
```

Comment:
seems like a copy of https://github.com/yast/yast-network/blob/87918f832546e76f124428b7efb9ca6b0789e877/src/lib/network/network_autoyast.rb#L382

Reply:
Seems it behaves the same way, yes.